run.sh

```diff
@@ -50,9 +50,6 @@ if [ "$INITIALIZE_TOOL_DEPENDENCIES" -eq 1 ]; then
     python ./scripts/manage_tool_dependencies.py init_if_needed
 fi
 
-[ -n "$GALAXY_UWSGI" ] && APP_WEBSERVER='uwsgi'
-find_server "${GALAXY_CONFIG_FILE:-none}" galaxy
-
 if [ "$run_server" = "python" -a -n "$GALAXY_RUN_ALL" ]; then
```

Comment:
`$run_server` cannot be `python` any more, so the `if` branch here can be dropped.